.buildkite/pipeline-resource-definitions/kibana-serverless-release.yml

```diff
@@ -25,7 +25,7 @@ spec:
       allow_rebuilds: false
       skip_intermediate_builds: false
       repository: elastic/kibana
-      pipeline_file: .buildkite/pipelines/serverless_deployment/run_serverless_release_assistant.yml
+      pipeline_file: .buildkite/pipelines/es_serverless/run_serverless_release_assistant.yml
```

Comment:
Not strongly opposed to this change if it makes sense to everybody else. However IMO `serverless_deployment` was a more intuitive place for this pipeline than `es_serverless` is, because the pipeline is kicking off the Kibana serverless release and is not related to ES serverless.

Reply:
oh, you're right, I have my wiring all crossed up in my head :P